tests/drum/test_inference.py

```diff
@@ -376,15 +376,18 @@ def test_custom_models_with_drum_nginx_prediction_server(
     @pytest.mark.parametrize(
         "framework, problem, language, docker, use_arrow",
         [
-            (SKLEARN_TRANSFORM_DENSE, TRANSFORM, PYTHON_TRANSFORM_DENSE, None, True),
-            (SKLEARN_TRANSFORM, TRANSFORM, PYTHON_TRANSFORM, None, False),
-            (SKLEARN_TRANSFORM_DENSE, TRANSFORM, PYTHON_TRANSFORM_DENSE, None, False),
+            # The following 3 tests produce Y transform values and are being temporarily removed until y transform
+            # validation is added
+            # (SKLEARN_TRANSFORM_DENSE, TRANSFORM, PYTHON_TRANSFORM_DENSE, None, True),
+            # (SKLEARN_TRANSFORM, TRANSFORM, PYTHON_TRANSFORM, None, False),
+            # (SKLEARN_TRANSFORM_DENSE, TRANSFORM, PYTHON_TRANSFORM_DENSE, None, False),
             (SKLEARN_TRANSFORM, TRANSFORM, PYTHON_TRANSFORM_NO_Y, None, True),
+            (SKLEARN_TRANSFORM, TRANSFORM, PYTHON_TRANSFORM_NO_Y, None, False),
```

Comment:
nit: we should refactor these constants as follows:
PYTHON/R_TRANSFORM -> implies there is no y
PYTHON/R_TRANSFORM_WITH_Y -> implies there is y

Reply:
I will work on refactoring that in a follow up https://datarobot.atlassian.net/browse/RAPTOR-6233